import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
```

Comment:
Hmm, yes I think you're right, there's not a lot of point having distkey be a unique column.  OTOH, I still feel like we're more likely to want query in a timestamp-driven fashion.